engines/scumm/scumm.cpp

```diff
@@ -312,7 +312,7 @@ ScummEngine::ScummEngine(OSystem *syst, const DetectorResult &dr)
 		_herculesBuf = (byte *)malloc(kHercWidth * kHercHeight);
 	}
 
-	_isRTL = (_game.version >= 4 && _game.version < 7 && _game.heversion == 0 && _language == Common::HE_ISR);
+	_isRTL = _game.id == GID_MANIAC || (_game.version >= 4 && _game.version < 7 && _game.heversion == 0 && _language == Common::HE_ISR);
```

Comment:
Please check it again, this change is definitely wrong

Reply:
Oops, thanks